src/styles/blocks/card.scss

```
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 30px 16px 16px;
+  margin: 50px 24px 0;
+  font-size: 10px;
+  line-height: 14px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $color-white;
+  color: $color-grin;
+}
+
+.card__title {
+  margin: 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: $color-dark;
+}
```

```suggestion
&__title {
  margin: 0;
  font-weight: 500;
  font-size: 12px;
  line-height: 18px;
  color: $color-dark;
}
```

and move it inside `.card` 
with '&' you can pass name of main selector used to write css
fix it everywhere please
you can move everything in this file inside `.card` using nesting